pcsx2/gui/MainFrame.cpp

```diff
@@ -610,20 +610,20 @@ void MainEmuFrame::ApplyCoreStatus()
 		if( !CoreThread.IsClosing() )
 		{
 			susres->Enable();
-			susres->SetItemLabel(_("Pause"));
+			susres->SetItemLabel(_("&Pause"));
 			susres->SetHelp(_("Safely pauses emulation and preserves the PS2 state."));
 		}
 		else
 		{
 			susres->Enable(vm);
 			if( vm )
 			{
-				susres->SetItemLabel(_("Resume"));
+				susres->SetItemLabel(_("Resu&me"));
```

Comment:
It probably would work better that way if you want to switch between pause and resume with they keyboard a lot. I've updated it.